purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -0,0 +1,85 @@
+package com.revenuecat.purchases
+
+import android.content.Context
+import java.util.concurrent.ExecutorService
+
+open class PurchasesConfiguration(builder: Builder) {
+
+    val context: Context
+    val apiKey: String
+    val appUserID: String
+    val observerMode: Boolean
+    val service: ExecutorService?
+    internal val store: Store
+    internal val diagnosticsEnabled: Boolean
+    val dangerousSettings: DangerousSettings
+    val verificationMode: EntitlementVerificationMode
+
+    init {
+        this.context = builder.context
+        this.apiKey = builder.apiKey
+        this.appUserID = builder.appUserID
+        this.observerMode = builder.observerMode
+        this.service = builder.service
+        this.store = builder.store
+        this.diagnosticsEnabled = builder.diagnosticsEnabled
+        this.verificationMode = builder.verificationMode
+        this.dangerousSettings = builder.dangerousSettings
+    }
+
+    open class Builder(
+        @get:JvmSynthetic internal val context: Context,
+        @get:JvmSynthetic internal val apiKey: String,
+        @get:JvmSynthetic internal val appUserID: String,
+    ) {
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var observerMode: Boolean = false
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var service: ExecutorService? = null
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var store: Store = Store.PLAY_STORE
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var diagnosticsEnabled: Boolean = false
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var verificationMode: EntitlementVerificationMode = EntitlementVerificationMode.default
+
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var dangerousSettings: DangerousSettings = DangerousSettings(customEntitlementComputation = true)
+
+        fun service(service: ExecutorService) = apply {
+            this.service = service
+        }
+
+        /**
+         * Sets the [EntitlementVerificationMode] to perform signature verification of requests to the
+         * RevenueCat backend.
+         *
+         * When changing from [EntitlementVerificationMode.DISABLED] to other modes, the SDK will clear the
+         * CustomerInfo cache.
+         * This means users will need to connect to the internet to get their entitlements back.
+         *
+         * The result of the verification can be obtained from [EntitlementInfos.verification] or
+         * [EntitlementInfo.verification].
+         *
+         * Default mode is disabled. Please see https://rev.cat/trusted-entitlements for more info.
+         */
+        fun entitlementVerificationMode(verificationMode: EntitlementVerificationMode) = apply {
+            this.verificationMode = verificationMode
+        }
```

Comment:
this shouldn't be added here, it's not a part of this mode

Reply:
I see you're already removing it in #1167